### ISSUE TITLE
Feature Request: add wp-cli support

### ISSUE THREAD

**Issue by COLABORATI** [bordoni/fakerpress#88]
yes, all the plugin does should be possible with wp-cli - and then it would be great to just load one saved setting and generate customized test data in one line: 
```
wp fakerpress --config=/path/to/config
```

**Comment by bordoni** [comment]
Yes that is also a feature that I've started. It's a big one, but I will do it via Administration first, then create a cool `cli` FakerPress command with all the things you can do on the Admin.

**Comment by curlup** [comment]
Any progress? Do you need any help?

**Comment by bordoni** [comment]
It would be nice to have some help, if you want to spin up a Pull Request or have any question in terms of which way to approach this @curlup, let me know here and we will get it done.

**Comment by nlemoine** [comment]
For people interested in a wp-cli fake data command line generator, I forked [Faker](https://github.com/trendwerk/faker) to support more content types: https://github.com/nlemoine/wp-cli-fixtures

**Comment by igrigorik** [comment]
Would love to see CLI support as well. Any progress or followup thoughts on this from maintainers?

**Comment by bordoni** [comment]
Working on it for the release `0.5.0`, which should include some actual refactoring of the code so we can have tests for the plugin running on Travis. Been honest, the release is still a bit far away.

**Comment by igrigorik** [comment]
@bordoni awesome, glad to hear it's on the roadmap! For context, we're investigating building an automated audit/benchmark environment to test WP themes and a CLI interface would be _hugely_ helpful as part of this flow. If there is a way to accelerate or uncouple this work from larger 0.5 release...

**Comment by bordoni** [comment]
@igrigorik probably I could decouple it for sure. Postponing the refactor of the base modules for tests to accomplish the CLI first doesn't seem bad at all.

So I will push the base for the WP-CLI integration to the `release/0.5.0` (this weekend).

I would love to be able to allow FakerPress to be the tool for your Benchmark env.

Let me know if you can share more of the specifics of the project with me later on so I can tailor the FakerPress stuff. `gustavo at bordoni.me`

**Comment by igrigorik** [comment]
@bordoni that's awesome! 

> So I will push the base for the WP-CLI integration to the release/0.5.0 (this weekend).

Just to clarify, you already have some WP-CLI integration code, which you're planning to merge into release/0.5.0 this weekend, or?  

Also, look for a Google Docs ping -- shared a WIP project draft with you. :-)

**Comment by bordoni** [comment]
I have some stuff done, but more related to the refactor, and  I would rather accomplish the CLI first. 

So the idea is taking a bit of time this weekend to merge a first draft with a few commands for CLI, to get the ball rolling.
